src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -53,11 +77,29 @@
             Ref("DoubleQuotedLiteralSegment"),
         ]
     ),
-    PostTableExpressionGrammar=Sequence(
```

Comment:
Why was this removed?

Reply:
> Why was this removed?

It wasn't needed after I expanded the scope for the INDEX functionality by added in the IndexHintClauseSegment at the bottom of the dialect definition so I removed this.